FrontEnd/src/components/ProfilePage/FormComponents/ProductServiceInfo.js

```diff
@@ -16,6 +18,32 @@ const ProductServiceInfo = (props) => {
     const { user } = useUser();
     const { profile: mainProfile, mutate: profileMutate } = useProfile();
     const [profile, setProfile] = useState(props.profile);
+    const { setFormIsDirty } = useContext(DirtyFormContext);
+
+    // TODO: update default values as new fields added
```

Comment:
Do we need this todo on all files?

Reply:
We haven't final profile model with all the fields still. New fields will be added soon or later. I think it's okay to leave this comment as a reminder that default values should be updated then.